optimizer-express/demo/proxy.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const express = require('express');
+const app = express();
+const AmpOptimizerMiddleware = require('amp-toolbox-optimizer-express');
+const ampOptimizer = require('amp-toolbox-optimizer');
+const compression = require('compression');
+const httpProxy = require('http-proxy');
+const https = require('https');
+const runtimeVersion = require('amp-toolbox-runtime-version');
+const PreloadImagesTransformer = require('./lib/PreloadImagesTransformer');
+const GoogleFontsPreconnectTransformer = require('./lib/GoogleFontsPreconnectTransformer');
+const apicache = require('apicache');
+
+const cache = apicache.middleware;
+
+// Configure the transformers to be used.
+// otherwise a default configuration is used.
+ampOptimizer.setConfig({
```

Comment:
It's better to not manually enumerate all transformers, instead add custom transformers to the default config.

Reply:
Enumerating provides better control on the insertion point of the transformer, if it has any dependency. I've created an issue to discuss how to improve the API here: https://github.com/ampproject/amp-toolbox/issues/35